types/react-datepicker/index.d.ts

```diff
@@ -198,6 +198,7 @@ export interface ReactDatePickerProps<
     excludeScrollbar?: boolean | undefined;
     enableTabLoop?: boolean | undefined;
     yearItemNumber?: number | undefined;
+    toggleCalendarOnIconClick?: boolean | undefined;
```

Comment:
How about moving this line below the `todayButton?` line and making it alphabetical?

Reply:
Thank you for the suggestion.  I moved the `toggleCalendarOnIconClick` below the `todayButton` prop.  Kindly verify @seckie